src/components/panel/panel.js

```diff
@@ -2209,24 +2397,31 @@ function MdPanelAnimation($injector) {
   this._$mdUtil = $injector.get('$mdUtil');
 
   /**
-   * @private {{element: !angular.JQLite|undefined, bounds: !DOMRect}|
-   *    undefined}
+   * @type {{element: !angular.JQLite|undefined, bounds: !DOMRect}|
+   * undefined}
+   * @private
```

Comment:
These should be combined. You don't need @private and @type. Just @private is preferred.